pkg/reconciler/v1alpha1/service/service_test.go

```diff
@@ -116,6 +116,33 @@ func TestReconcile(t *testing.T) {
 			Eventf(corev1.EventTypeNormal, "Created", "Created Configuration %q", "pinned2"),
 			Eventf(corev1.EventTypeNormal, "Created", "Created Route %q", "pinned2"),
 		},
+	}, {
+		Name: "pinned - with ready config and route",
+		Objects: []runtime.Object{
+			svc("pinned3", "foo", WithReleaseRollout("pinned3-0001"),
+				WithInitSvcConditions),
+			config("pinned3", "foo", WithReleaseRollout("pinned3-0001"), WithGeneration(1),
+				WithLatestCreated, WithObservedGen,
+				// When we see the LatestCreatedRevision become Ready, then we
+				// update the latest ready revision.
```

Comment:
I'd drop this, generally this style of comment is to indicate the delta in the `WantUpdates` section.

Reply:
/done